docs/specs/bug_and_feedback_submission.md

```diff
@@ -35,12 +35,24 @@ Cons:
 
 ## Solution
 For the first iteration we'll use the same setup as with Analytics. We'll setup four new paths on the `data.trezor.io` domain:
-- `/suite/feedback/develop.log`
-- `/suite/feedback/prod.log`
-- `/suite/bugs/develop.log`
-- `/suite/bugs/prod.log`
 
-with the same exact logging configuration. Tomas ferko will re-use the python scripts in Keboola.
+For production builds:
+- `/suite/feedback/web/stable.log`
+- `/suite/feedback/desktop/stable.log`
+- `/suite/feedback/mobile/stable.log`
+- `/suite/bugs/web/stable.log`
+- `/suite/bugs/desktop/stable.log`
+- `/suite/bugs/mobile/stable.log`
+
+Other usage (localhost, sldev, etc.):
+- `/suite/feedback/web/develop.log`
+- `/suite/feedback/desktop/develop.log`
+- `/suite/feedback/mobile/develop.log`
+- `/suite/bugs/web/develop.log`
+- `/suite/bugs/desktop/develop.log`
+- `/suite/bugs/mobile/develop.log`
```

Comment:
What's the reason to distinguish platforms on the URL level? Having several URLs makes both the infra and client code more complex.

Reply:
It's now aligned with Analytics, it's true in case of feedback it might be enough to have it in the data only, I'll simplify it. What do you think @tomasklim ?

Reply:
I agree to have it only in data. 

I am not sure why analytics distinguish it. Thx, good point for analytics 2.0.

Reply:
👍